PWGGA/GammaConv/macros/AddTask_TRDEfficiency.C

```diff
@@ -38,7 +38,61 @@ AliAnalysisTRDEfficiency* AddTask_TRDEfficiency(
 
     // ================== GetInputEventHandler =============================
     AliVEventHandler *inputHandler=mgr->GetInputEventHandler();
-    
+    cout << "$$$$$$$$$ part of the task $$$$$$$$" << endl;
+    //=========  Set Cutnumber for V0Reader ================================
+    TString cutnumberPhoton = "00200009227300008250400000";
+    TString cutnumberEvent = "00010113";
+    AliAnalysisDataContainer *cinput = mgr->GetCommonInputContainer();
+    //========= Add V0 Reader to  ANALYSIS manager if not yet existent =====
+    TString V0ReaderName = Form("V0ReaderV1_%s_%s",cutnumberEvent.Data(),cutnumberPhoton.Data());
+    if( !(AliV0ReaderV1*)mgr->GetTask(V0ReaderName.Data()) ){
+        AliV0ReaderV1 *fV0ReaderV1 = new AliV0ReaderV1(V0ReaderName.Data());
+        cout << "@@@@@@@@@  in the movement  @@@@@@@@@" << endl;
+        fV0ReaderV1->SetUseOwnXYZCalculation(kTRUE);
```

Comment:
if there are no mistakes in the setup, everything within the if-statement should never be run. you may consider to drop it and issue a warning if the V0reader cannot be found - you may also check the GammaConvCalo_pp AddTask for example